docs/content/ecs/component-generation.md

```diff
@@ -13,12 +13,28 @@ The [code generator]({{urlRoot}}/content/code-generator) uses `.schema` files to
 
 ## Overview
```

Comment:
Just noticed this doc is in ECS directory. TBH, the directory structure is a bit of a mess due to Docs-mageddon. Will add it to my "fiddle about with things" task.